MigrationCodelab/build.gradle

```diff
@@ -24,7 +24,7 @@ buildscript {
 
         // App dependencies
         appCompatVersion = '1.1.0'
-        composeVersion = '1.0.0-SNAPSHOT'
+        composeVersion = '1.0.0-alpha05'
         constraintLayoutVersion = '2.0.0-rc1'
```

Comment:
ConstraintLayout is already stable: `"androidx.constraintlayout:constraintlayout:2.0.2"` https://developer.android.com/jetpack/androidx/releases/constraintlayout

Reply:
Done! Thanks